### ISSUE TITLE
Field name toLowerCase

### ISSUE THREAD

**Issue by 7img** [pburtchaell/react-input#4]
In the Forms component the prop fields key is changed by a toLowerCase. By lowercasing the input names I understand that you prevent duplicate values in your object. However I'm working with an API that is case sensitive and requires camel casing. Of course I can do some sort of mapping and change the key's back to their original state with the input value but I'm wondering if this is really necessary. 

In the future I might want to add a translation to my React Application. Maybe its also an idea to add an extra field for the label text. If this is not set but label is set to true it can always use the field name as a fallback.

I wouldn't mind making the changes myself with a PR.

**Comment by pburtchaell** [comment]
I would agree, changing the props field key is not helpful. I've run into the same issue on projects I use this and in order to get around it I map the vales, but you are right, it isn't necessary. I certainly welcome a PR if you have time! 😄

**Comment by pburtchaell** [comment]
This is resolved in version 4.0.0. Instead of using the `name` prop, I added `key` prop. The name is used for the label and the key to ensure fields are unique.

```js
<Form
  fields={[{
    name: 'First name',
    key: 'firstname'
  }
  ...
>
